### PR TITLE
New graphql interface

### PR DESCRIPTION
**Pull request by aalmazan**
* Use [graphql-python/gql](https://github.com/graphql-python/gql)  for all GraphqQL usage
* Keeps existing API methods usable with the same parameters. Debug/logging behavior may change.
* In general, will solve multipart request handling (#14)
* First pass will exclusively use `requests`. A future pass will decouple this and allow the user to provide their own GQL transport (https://gql.readthedocs.io/en/latest/transports/index.html) to allow async. 

~~Partially blocked by: https://github.com/graphql-python/gql/issues/385~~
~~`gql` fork may be used instead for a while until an update is made.~~

### CODE REVIEW

**Review by fostimus — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# 2.1.0 (2023-02-17)
```
Comment (benogle) [inline CHANGELOG.md:1]: This definitely should be a major revision!

Comment (benogle) [reply CHANGELOG.md:1]: Is this already published as 2.1?

File: examples/make_graphql_request.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+              }
+              welds {
+                eid
+                name
+              }
+            }
+          }
+        }
+    """
+    res = anvil.query(query=user_query, variables=None)
```
Comment (benogle) [inline examples/make_graphql_request.py:41]: Should these examples use the new query builder pattern?

**Review by benogle — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (aalmazan) [reply CHANGELOG.md:1]: Nope, just getting it ready. But yeah it could be a 3.0.

**Pull request merged**